app/models/supervisor_script_executors/supervisor_script_executors.rb

```diff
@@ -0,0 +1,33 @@
+Dir[Rails.root.join('supervisor_scripts', 'executors', '*_executor.rb').to_s].each {|file| require file}
+require_relative 'abstract_supervisor_script_executor'
+##
+# This class translates simulation scripts id to their executor
+class SupervisorScriptExecutors
+  private
```

Comment:
poza tym jest OK, dalsze poprawki i przemyślenia na temat tego jak ma wyglądać interfejs executora będą w ramach zadania SCAL-690; zmerguję jak poprawisz to private